qiskit/algorithms/__init__.py

```diff
@@ -96,17 +114,6 @@
    VQD
    VQDResult
 
-Eigensolvers
-------------
-
-Algorithms to find eigenvalues of an operator. For chemistry these can be used to find excited
-states of a molecule, and qiskit-nature has some algorithms that leverage chemistry specific
-knowledge to do this in that application domain.
-
-.. autosummary::
-   :toctree: ../stubs/
-
-   eigensolvers
 
 Evolvers
 --------
```

Comment:
I like your proposal, but I think that then the evolvers should follow the same format. Have the legacy time evolvers and then the new modules. I find it a bit strange to have this link exclusively here.

Reply:
Done in 40fea8b